docs/articles/posts/20250416_joseph_wilson_interview.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Interview with a Data Scientist - The Aerospace Engineer who Loved Spreadsheets
+authors: [josephwilson]
+date: 2025-04-16
+links:
+    - https://nhsengland.github.io/datascience-pd-newsletter/posts/2025_03/newsletter.html
+slug: interview_with_joseph_wilson
```

Comment:
looks great! only thing: do you not wanna add any categories on it?

Reply:
A second attempt to add categories has worked! I've tweaked some other articles' categories to make it more consisent